corehq/apps/hqwebapp/views.py

```diff
@@ -727,125 +849,27 @@ def post(self, req, *args, **kwargs):
-            ))
 
-            debug_context.update({
-                'self_started': domain_object.internal.self_started,
-                'has_handoff_info': bool(domain_object.internal.partner_contact),
-                'project_description': domain_object.project_description,
-            })
-
-        subject = '{subject} ({domain})'.format(subject=report['subject'], domain=domain)
+@method_decorator([login_required], name='dispatch')
+class SolutionsFeatureRequestView(View):
+    urlname = 'solutions_feature_request'
 
-        if full_name and not any([c in full_name for c in '<>"']):
-            reply_to = '"{full_name}" <{email}>'.format(**report)
-        else:
-            reply_to = report['email']
-
-        # if the person looks like a commcare user, fogbugz can't reply
-        # to their email, so just use the default
-        if settings.HQ_ACCOUNT_ROOT in reply_to:
-            reply_to = settings.SERVER_EMAIL
-
-        message += "Message:\n\n{message}\n".format(message=report['message'])
-        if post_params.get('five-hundred-report'):
-            extra_message = ("This message was reported from a 500 error page! "
-                             "Please fix this ASAP (as if you wouldn't anyway)...")
-            extra_debug_info = (
-                "datetime: {datetime}\n"
-                "Is self start: {self_started}\n"
-                "Has Support Hand-off Info: {has_handoff_info}\n"
-                "Project description: {project_description}\n"
-                "Sentry Error: {sentry_error}\n"
-            ).format(**debug_context)
-            traceback_info = cache.cache.get(report['500traceback']) or 'No traceback info available'
-            cache.cache.delete(report['500traceback'])
-            message = "\n\n".join([message, extra_debug_info, extra_message, traceback_info])
-
-        email = EmailMessage(
-            subject=subject,
-            body=message,
-            to=[settings.SUPPORT_EMAIL],
-            headers={'Reply-To': reply_to},
-            cc=other_recipients
+    def post(self, request, *args, **kwargs):
+        email = _get_email_message_base(
+            post_params=request.POST,
+            couch_user=request.couch_user,
+            uploaded_file=request.FILES.get('feature_request'),
+            to_email=settings.INTERNAL_FEEDBACK_EMAIL,
         )
-
-        if uploaded_file:
-            filename = uploaded_file.name
-            content = uploaded_file.read()
-            email.attach(filename=filename, content=content)
-
-        # only fake the from email if it's an @dimagi.com account
-        is_icds_env = settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS
-        if is_dimagi_email(report['username']) and not is_icds_env:
-            email.from_email = report['username']
-        else:
-            email.from_email = settings.SUPPORT_EMAIL
-
-        return email
+        email.send(fail_silently=False)
```

Comment:
Might be good to tell the user all went well?

Reply:
After adding the missing JS files ([1b19cdf](https://github.com/dimagi/commcare-hq/pull/34026/commits/1b19cdf9ab50b6d12a666e4b8256751a0cd6b3ab)), the user should now see a success message as follows:
![Screenshot from 2024-01-25 12-23-39](https://github.com/dimagi/commcare-hq/assets/122617251/e95f5271-5bd4-472c-897e-2dc628a2d346)